### DISCUSSION TITLE
Building latest kong (3.6.x) on alpine base image

### DISCUSSION THREAD

**Discussion by alokrbl** [Kong/kong#13480] [Help] [answered]
I’m aware that support for the Alpine base image and Alpine packages was discontinued with Kong 3.4. 
Can one still construct a Kong 3.6 image using an Alpine base and install Kong packages on it, perhaps using a deb package? Additionally, what prompted the removal of Alpine support? 
Utilizing an Ubuntu base image to build Kong results in a significantly larger Docker image than when using Alpine.

Thanks in advance for the support.

**Comment by ProBrian** [comment]
Have you tried building your custom package based on alpine? refer to https://docs.konghq.com/gateway/latest/install/linux/debian/#package-install to install via deb package.

**Comment by chronolaw** [comment]
See this PR: https://github.com/Kong/kong/pull/10926

**Comment by alokrbl** [comment]
@chronolaw, Thanks for the response.
 I know that the alpine support has been removed but I am unable to find the reason for removing alpine support.  were there any security/vulnerability or some technical challenge?

Also, If we go with alpine variant, Are there any feedback of its outcome and challenges?
We are trying to run Kong, by building docker image ( based on alpine) using Kong deb packages

Thanks in advance!

**Reply by chronolaw** [reply, marked as answer]
A short and simple answer:
- Alpine was never recommended for production use.
- Alpine upstream either not ships with key dependencies or ships versions that are too old for Kong. 

We try to focus on ‘only’ production platform support.
